packages/form-builder/src/components/FormField/mixins/choiceComponent.ts

```diff
@@ -1,36 +1,57 @@
+import Vue, { PropType } from 'vue';
 import Component, { mixins } from 'vue-class-component';
 
-import { ChoiceValue, FieldItem, FieldItemValue } from '../types';
-
-import { FieldComponent } from './fieldComponent';
+import { ChoiceFieldValue, FieldItem, FieldOptions, FieldItemValue } from '../types';
+
+const Props = Vue.extend({
+	props: {
+		/** The choice field value to display */
```

Comment:
Du coup ce n'est plus vraiment "to display" !
```suggestion
		/** The field value */
```